app/services/one_signal_service.rb

```diff
@@ -0,0 +1,20 @@
+class OneSignalService
```

Comment:
As you don't need to have an instance of this class, you can change this into a `module` and extend self to declare the class methods (if you have any doubts, it's specified in the rootstrap rails guides how to do this). This way when you call the OneSignal  service you wont need to use `.new`

Reply:
Also, there is a OneSignal gem available, anyway you can keep this. It's just a comment to let you know its existance